### ISSUE TITLE
Completely shut down FC1

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#1527]
needs #931
Any other requirements?

**Comment by Captainkirkdawson** [comment]
Should be in FC

**Comment by richpomfret** [comment]
Closing as not applicable.
